### ISSUE TITLE
On Contact does not work on animated objects parented to a scaled object

### ISSUE THREAD

**Issue by blackno666** [armory3d/armory#1299]
v2019.6 ($Id: a1f62c5c53fd416d5483932a490fe041c6af29f3 $)

I have an animated cube and a player. On contact the cube turns red.
I then have the same cube parented to an empty which has a scale, but for the collision physics, the scale of the parent is not taken into consideration, leading to unexpected results:

![collsion](https://user-images.githubusercontent.com/47137964/59802304-088cef00-92e9-11e9-896b-d1de3b23f2f9.gif)

The middle cube is the player. The right cube acts as expected; the left cube is parented to a scaled empty and does not behave as expected.

Here's the .blend: [Collision.zip](https://github.com/armory3d/armory/files/3307949/Collision.zip)

Basically, what I want to achieve is to have the same animation applied to objects of different size. Since Delta Transforms don't work (#1240) I thought using a parent might solve this issue, but alas ;)
